R/generate.R

```diff
@@ -3,6 +3,8 @@
 #' @param reps the number of resamples to generate
 #' @param type currently either \code{bootstrap}, \code{permute}, or \code{simulate}
 #' @param ... currently ignored
+#' @return A tibble containing a \code{replicate} column and corresponding \code{stat} column 
```

Comment:
Was this copy + pasted from `calculate()`? Should be

A tibble containing \code{rep} generated datasets, indicated by the \code{replicate} column.

Reply:
The dangers of copy and paste. Updating now.